linode_api4/groups/linode.py

```diff
@@ -307,10 +308,15 @@ def instance_create(
             kwargs["firewall_id"] = fw.id if isinstance(fw, Firewall) else fw
 
         if "interfaces" in kwargs:
-            kwargs["interfaces"] = [
-                i._serialize() if isinstance(i, ConfigInterface) else i
-                for i in kwargs["interfaces"]
-            ]
+            interfaces = kwargs["interfaces"]
+            kwargs["interfaces"] = (
+                [
+                    i._serialize() if isinstance(i, ConfigInterface) else i
+                    for i in interfaces
+                ]
+                if isinstance(interfaces, Iterable)
+                else interfaces
+            )
```

Comment:
optional: I think we can simplify this down a bit

```suggestion
        interfaces = kwargs.get("interfaces")
        if interfaces is not None and isinstance(interfaces, Iterable):
            kwargs["interfaces"] = [
                i._serialize() if isinstance(i, ConfigInterface) else i
                for i in interfaces
            ]
```

Reply:
Thanks!